cells/patroni/nomadCharts/srv-rest.nix

```diff
@@ -21,16 +21,5 @@
     "\${NOMAD_ALLOC_ID}"
     namespace
     "ingress"
```

Comment:
This "ingress" tag should be deleted as there is no longer an ingress with the tags below being deleted.

Reply:
(Manveru already mentioned this one above I just noticed)